docs/reference/commandline/dockerd.md

```diff
@@ -1,16 +1,12 @@
-<!--[metadata]>
-+++
-title = "dockerd"
-aliases = ["/engine/reference/commandline/daemon/"]
-description = "The daemon command description and usage"
-keywords = ["container, daemon, runtime"]
-[menu.main]
-parent = "smn_cli"
-weight = -1
-+++
-<![end-metadata]-->
-
-# daemon
+---
+redirect_from:
+- /reference/commandline/dockerd/
```

Comment:
oh, I guess we need `/reference/commandline/daemon/` here (instead or "as well"), I don't think we ever had `/reference/commandline/dockerd/`